css/selectors/user-invalid.json

```diff
@@ -15,12 +15,24 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "4"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox": [
+              {
+                "version_added": false
```

Comment:
I think we don't need multiple statements here. See also https://github.com/mdn/browser-compat-data/issues/7404

Basically, `"version_added": "4"` and `alternative_name": "::-moz-ui-invalid"` already indicate that it is only supported this way and that there is no standard support for `:user-invalid`.